libcxx/include/__chrono/tzdb_list.h

```diff
@@ -32,9 +32,16 @@ _LIBCPP_BEGIN_NAMESPACE_STD
 
 namespace chrono {
 
+// TODO TZDB
+// Libc++ recently switched to only export __ugly_names from the dylib.
+// Since the library is still experimental the functions in this header
+// should be adapted to this new style. The other tzdb headers should be
+// evaluated too.
+
 class _LIBCPP_AVAILABILITY_TZDB tzdb_list {
 public:
-  _LIBCPP_EXPORTED_FROM_ABI explicit tzdb_list(tzdb&& __tzdb);
+  class __impl; // public to allow construction in dylib
+  _LIBCPP_EXPORTED_FROM_ABI explicit tzdb_list(__impl* __p);
```

Comment:
Do we really need this in the dylib too? I think the type only needs to be complete at the point where we destroy the `unique_ptr`, so I think this could be made `HIDE_FROM_ABI`.

Reply:
Thanks for catching this one.